cypress/e2e/cypressActions.cy.js

```diff
@@ -0,0 +1,40 @@
+/// <reference types='cypress' />
+
+const { generateUser } = require('../support/generate');
+
+describe('Name of the group', () => {
```

Comment:
name your describe more appropriately, and the name of your `it` should continue the describe, e.g
`describe('User should be able')  
	it('to sign in with valid data')
`